lib/build_pack/downloader.rb

```diff
@@ -0,0 +1,32 @@
+module BuildPack
+  class Downloader
+    MYSQL_BASE_URL = "http://security.debian.org/pool/updates/main/m/mysql-5.5/"
+    REGEX = /.*(mysql-client-5\.5_5\.5\...-0\+deb.u._amd64.deb).*/
```

Comment:
What would happen if this jumps to `mysql-client-5.5_5.5.6x-0+deb8u1_arm64.deb`?